### ISSUE TITLE
Greeter displays user's wallpaper to other people

### ISSUE THREAD

**Issue by HansCronau** [elementary/greeter#299]
This is a UX remark in response to the [call for testing on Reddit](https://www.reddit.com/r/elementaryos/comments/c68q3u/call_for_testing_the_new_login_lock_screen/).

As a user picking a desktop wallpaper, I ~am currently~ used to be concerned with only one question:

- **How does the wallpaper look on my desktop?**

Since the new greeter shows users' wallpapers behind their avatar new considerations are introduced to my choice of wallpaper:

- **Is the wallpaper aesthetically pleasing with my avatar in front of it?** Taking this question serious may be ruling out many wallpapers. Some will be too detailed compared to the avatar. Others may have important details in the spot where the avatar's circle would go. Another issue may be that those who would want to check for this would have to through the steps: change wallpaper, lock account, check looks, log back in, and repeat.
- **How do I feel about this wallpaper representing me?**
  - This may be especially relevant since the wallpaper claims more screen space than the actual avatar the user has already chosen to represent them.
  - ~Users who would have previously chosen a very personal or otherwise sensitive image for a wallpaper may now feel uncomfortable doing so.~ As @cassidyjames points out below, this would have been an issue with the old Greeter as well.
  - Users who used to have their desktop display a picture of a loved one, now have to choose if they want the face of _another person_ displayed to represent them, possibly claiming more screen space than their own face within the avatar.
- **Should I now change my wallpaper as little as possible?** Users who want to keep their appearance consistent to others may now feel reluctant to change their wallpaper too often. This may be a concern especially to those who would otherwise like to set up a (live) wallpaper that changes every hour.

Because of Twitter and Facebook some users may not be new to the idea of choosing an image to go with/behind their avatar. This is the first time however, that I have seen it combined with a choice of wallpaper. Because I believe each application of the wallpaper image has its own set of considerations, I would suggest an option to disable the extra picture behind the avatar, and/or an option for users to choose it separately from their wallpaper. (Ideally the latter would include a preview that also shows the avatar, though that would require more extensive changes.)

**Comment by HansCronau** [comment]
I was asked for a reminder on the current lock screen's use of user wallpapers for those who have been testing the new Greeter for some time now.

As far as I can tell, the current lock screen does not display any user wallpapers. It displays users with their avatar and name only. It does have a background, for which it uses the default Pablo Garcia Saldana wallpaper. If you remember the Greeter's background matching your wallpaper, you may have achieved this through [some hacking](https://elementaryos.stackexchange.com/questions/81/how-do-i-change-login-background-in-elementary-os-freya-or-loki). I must note that [some comments](https://elementaryos.stackexchange.com/questions/81/how-do-i-change-login-background-in-elementary-os-freya-or-loki#comment21880_120) in that thread remark that the Greeter _was_ supposed to switch between user wallpapers. If that is indeed true, it may not have shown user wallpapers not by design but because of a broken feature.

Whatever the case, I don't believe the lock screen's past looks (or the looks of any OS's lock screen for that matter) are of huge influence to the considerations I have listed above. I believe those are mainly a result of using user wallpapers as a social media style rectangle behind their avatars.

**Comment by cassidyjames** [comment]
The current stable Greeter does indeed show user wallpapers. This is not a functional change.

**Comment by HansCronau** [comment]
@cassidyjames Thanks for the clarification!

In that case any worries about wallpapers being inappropriate for the Greeter would not be new.

**Comment by HansCronau** [comment]
Here's a (very high tech) screenshot that illustrates some of the points above.

![screenshot](https://user-images.githubusercontent.com/977860/60358444-9c377d00-99c5-11e9-803c-7ed53ff6dfd8.png)

On the left we see Forrest. Or... actually mostly his girlfriend, Jewel.
Then there's me logging in and next to me I notice Peter's a fan of anime. Initially I thought he would be ashamed to know that I had seen his wallpaper, but it was just pointed out by @cassidyjames that this is not new, so Peter probably doesn't mind.
On the far right there's Thelma. She has a really wallpaper of her car, but sadly it doesn't work as well for the login screen.

**Comment by cassidyjames** [comment]
GNOME appears to be addressing this with a sort of mockup of the wallpaper on the greeter within the settings pane.

![screenshot](https://gitlab.gnome.org/GNOME/gnome-control-center/uploads/ea4b74f293d779b621f3e4e21bde4a8b/new_panel.png)

If that's an approach we'd want to take, this issue should be moved to [elementary/switchboard-plug-pantheon-shell](https://github.com/elementary/switchboard-plug-pantheon-shell/). Thoughts from @elementary/ux?

**Comment by stsdc** [comment]
Such login cards always look great in mockups, because designer uses photos made by pros for avatars and curated backgrounds. 

There are always a typical solutions like blur wallpaper for login card background or extract main colours from wallpaper and make an "interesting" gradient.

**Comment by danrabbit** [comment]
There is already an issue opened here: https://github.com/elementary/switchboard-plug-pantheon-shell/issues/28

Close as duplicate?

**Comment by HansCronau** [comment]
@danrabbit That issue does adresses an important part of my suggestion: the option to change the lock screen image separately.

I think we can close this one as a duplicate.

**Comment by cassidyjames** [comment]
@danrabbit well, I know that's a feature request but what I was specifically addressing here was a way to inform users their picked wallpaper will show up on the lockscreen as well, along with trying to address the specific complaints raised here about not knowing how it will look. So that's separate from the ability to set a completely different wallpaper.

**Comment by HansCronau** [comment]
Fair enough: one could indeed build a separate lock screen image feature without visual feedback (when changing wallpapers) and one could provide visual feedback without allowing separate images to be set.

Should this issue be moved to https://github.com/elementary/switchboard-plug-pantheon-shell/ either way?
